### PR TITLE
update info

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4478

Besides what was requested, i also changed the parameters values to Boolean, as stated in the documentation for this cmdlet: https://docs.microsoft.com/en-us/powershell/module/skype/set-csaccessedgeconfiguration?view=skype-ps

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/configure-federation-with-skype-for-business-online.md [hunk 1]
```diff
@@ -24,22 +24,24 @@ description: "Summary: Learn how to configure interoperability between your on-p
 
 To configure Skype for Business hybrid, you need to:
 
-- [Configure your on-premises environment service to federate with Office 365](#configure-your-on-premises-edge-service-to-federate-with-office-365).
+- [Configure your on-premises environment service to federate with Office 365 or another organisation](#configure-your-on-premises-edge-service-to-federate-with-office-365).
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-federation-with-skype-for-business-online.md:27]:
```suggestion
- [Configure your on-premises Edge service to federate with Office 365 or another organization](#configure-your-on-premises-edge-service-to-federate-with-office-365-or-another-organization).
```

File: Skype/SfbHybrid/hybrid/configure-federation-with-skype-for-business-online.md [hunk 2]
```diff
@@ -24,22 +24,24 @@ description: "Summary: Learn how to configure interoperability between your on-p
...
 - [Configure your on-premises environment to trust Office 365 and enable shared SIP address space with Office 365](#configure-your-on-premises-environment-to-enable-shared-sip-address-space-with-office-365).
 - [Enable shared SIP address space in your Office 365 tenant](#enable-shared-sip-address-space-in-your-office-365-tenant).
 
 Note that if you have Exchange on-premises, you may want to configure OAuth between your Exchange on-premises and Skype for Business Online environments. For more information, see  [Manage server-to-server authentication in Skype for Business Server](https://docs.microsoft.com/SkypeForBusiness/manage/authentication/server-to-server-and-partner-applications) and [Plan to integrate Skype for Business and Exchange](https://docs.microsoft.com/SkypeForBusiness/plan-your-deployment/integrate-with-exchange/integrate-with-exchange#feature_support). 
   
-## Configure your on-premises Edge service to federate with Office 365
+## Configure your on-premises Edge service to federate with Office 365 or another organisation
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-federation-with-skype-for-business-online.md:33]:
```suggestion
## Configure your on-premises Edge service to federate with Office 365 or another organization
```

File: Skype/SfbHybrid/hybrid/configure-federation-with-skype-for-business-online.md [hunk 3]
```diff
@@ -24,22 +24,24 @@ description: "Summary: Learn how to configure interoperability between your on-p
...
+Set-CSAccessEdgeConfiguration -AllowOutsideUsers $True -AllowFederatedUsers $True -EnablePartnerDiscovery $True -UseDnsSrvRouting
 ```
 
-If '-EnablePartnerDiscovery' value set to 1, Skype for Business Server will use DNS records to try and discover partner domains not listed in the AllowedDomains list. If the value set to 0, Skype for Business Server will only federate with domains found on the AllowedDomains list. This parameter is required if you use DNS service routing.
+If '-EnablePartnerDiscovery' value is set to $True, Skype for Business Server will use DNS records to try and discover partner domains not listed in the AllowedDomains list. If the value is set to $False , Skype for Business Server will only federate with domains found on the AllowedDomains list. This parameter is required if you use DNS service routing.
 
+> [!NOTE]
+> For more details about enabling federation between users of your on-premises Skype for Business deployment and users of a Skype for Business Online organisation, check [this article](https://docs.microsoft.com/en-us/skypeforbusiness/manage/federation-and-external-access/federation-support/configuring-federation-support).
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/configure-federation-with-skype-for-business-online.md:44]:
```suggestion
> For more details about enabling federation between users of your on-premises Skype for Business deployment and users of a Skype for Business Online organization, see [Configuring federation support for a Skype for Business Online customer in Skype for Business Server](https://docs.microsoft.com/skypeforbusiness/manage/federation-and-external-access/federation-support/configuring-federation-support).
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**